main.go

```diff
@@ -369,7 +369,8 @@ func main() {
 
 // getManager return a controller Manager object that watches on the managed cluster and has the controllers registered.
 func getManager(
-	mgrCtx context.Context, options manager.Options, healthAddr string, hubCfg *rest.Config, managedCfg *rest.Config,
+	mgrCtx context.Context, options manager.Options, healthAddr string,
+	hubCfg *rest.Config, managedCfg *rest.Config, disableGk bool,
```

Comment:
The rest of this function uses the global values from `tool.Options` so you could do the same here.

Reply:
Ah, I didn't notice that was global.